sections/product_outline.liquid

```diff
@@ -1,8 +1,8 @@
 {% for category in current_product.outline.categories %}
   {% if category.pending_drip? %}
-    <h4>{ category.title }} - Drip</h4>
+    <h4>{{ category.title }} - Drip</h4>
```

Comment:
Looks like another PR snuck in here. 
I would copy product_outline from master and go to this branch and replace and commit it will remove this from the PR